bootloader/prebootloader/src/rtl872x/part1/sleep_handler.cpp

```diff
@@ -249,8 +249,31 @@ void configureDeepSleepWakeupSource(const hal_sleep_config_t* config) {
 
 // Copy and paste from SOCPS_DeepSleep_RAM()
 void enterDeepSleep() {
+#if PLATFORM_ID == PLATFORM_P2
```

Comment:
Do you need these #if checks at all? Shouldnt this apply to all RTL platforms like trackerm and msom as well?

Reply:
Ran the same test app against the MSOM branch, which has not yet included this changes, the D7 is always low when it is configured as output low and SWD. So it shouldn't have this issue.

Reply:
Weird. So this is a difference between RTL8721 and RTL8722? 
So far I have treated them as essentially identical, but perhaps there are differences like this behavior.

Reply:
Have you tested what MSOM does when the SDK change is applied? 
I'm worried about if we need to support different SDK configurations based on our hardware platforms.

Reply:
Just tested msom again when only include the SDK change is applied against the public `feature/add-msom-platform` branch, so both RTL8721 and RTL8722 behave the same. Whereas the changes to the pre-bootloader is just to make sure the D7 on Photon2 is not leaking power. MSoM doesn't connect a LED on SWD-DAT, so we don't need to include the changes.